.github/workflows/backend.yml

```diff
@@ -0,0 +1,81 @@
+name: Backend CI
+
+on:
+  push:
+    paths:
+      - app/proto/**
+      - app/backend/**
+      - app/proxy/**
+      - deployment/backend/**
+      - .github/workflow/backend.yml
+      - .github/actions/**
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+
+  build:
+    runs-on: [self-hosted, master]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/build-push-ecr
+        with:
+          module-name: backend
+
+  build-proxy:
+    uses: ./.github/workflows/proxy.yml
+
+
+  test-backend:
+    runs-on: [self-hosted, master]
+    needs: build
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/pre-tasks
+      - working-directory: app/backend
+        run: |
+          curl -sLO https://github.com/Couchers-org/resources/raw/a664750dc60771ec5080ac4753d9a27e6ec544b9/timezone_areas/timezone_areas.sql.zst
+          zstd -d timezone_areas.sql.zst -o resources/timezone_areas.sql
+          docker-compose -f ../docker-compose.test.yml up -d postgres_tests
```

Comment:
Possible perhaps? I've not had it happen yet>
Again because this is a temp solution to stand up the old backend.
We could just add a sleep? Installing requirements below takes a while before tests start